services/ext/api.go

```diff
@@ -1264,6 +1264,14 @@ func (api *PublicAPI) UpdateCommunityTokenState(contractAddress string, deploySt
 	return api.service.messenger.UpdateCommunityTokenState(contractAddress, deployState)
 }
 
+func (api *PublicAPI) ToggleCollapsedCommunityCategory(request *requests.ToggleCollapsedCommunityCategory) error {
+	return api.service.messenger.ToggleCollapsedCommunityCategory(request)
+}
+
+func (api *PublicAPI) CollapsedCommunityCategories() ([]protocol.CollapsedCommunityCategory, error) {
+	return api.service.messenger.CollapsedCommunityCategories()
```

Comment:
When I first read `CollapsedCommunityCategories` I thought the endpoint was returning only the records where `Collapsed` is true. Maybe more correct, but also longer, is to refer to these entities as *collapsible*, not *collapsed*, since they can be either one of the two collapsed or non-collapsed states. If you think this is a valid suggestion, then consider renaming the table and all other variables/functions too so that there's no confusion about the state of the record and the name of the entity.

Reply:
Actually, in theory we don't need to store non-collapsed categories, so you are making a very good point, we could delete them from the database if `collapsed` is `false`, I will change that, and then the endpoint name makes sense, what do you think?

Reply:
You're completely right, I missed that we don't need to store them.